README.md

```diff
@@ -1,17 +1,13 @@
-Faker
+factory-helper
 =====
-This gem is a port of Perl's Data::Faker library that generates fake data.
+This gem began as a fork of stympy's Faker which is a port of Perl's Data::Faker library that generates fake data.
 
-It comes in very handy for taking screenshots (taking screenshots for my
-project, [Catch the Best](http://catchthebest.com/) was the original impetus
-for the creation of this gem), having real-looking test data, and having your
-database populated with more than one or two records while you're doing
-development.
+Its primary purpose is to facilitate software testing by allowing factories to have a solid source of random data with defined characteristics, but can be used for many other purposes as well as in production code.
```

Comment:
At least at this early stage, it may be a good idea to spell out our reasons for creating a separate gem, i.e. addition of new features, many of which have been stuck in PRs on the original repo.